summary_workflows/quantification/main.nf

```diff
@@ -3,23 +3,26 @@
 if (params.help) {
 	
 	    log.info"""
-	    ==============================================
-	    TCGA CANCER DRIVER GENES BENCHMARKING PIPELINE 
-		Author: Javier Garrayo Ventas
-		Barcelona Suercomputing Center. Spain. 2019
-	    ==============================================
+	    =====================================================
+	    APAeval QUANTIFICATION BENCHMARKING PIPELINE
+		Author(s): Yuk Kei Wan (*1,*2), Asier Gonzalez (*3)
+		*1 iRNA COSI
+		*2 Genomic Institute of Singapore, A*STAR, Singapore
+		*3 Barcelona Supercomputing Center, Barcelone, Spain
+		2021
+	    =====================================================
 	    Usage:
 	    Run the pipeline with default parameters:
 	    nextflow run main.nf -profile docker
```

Comment:
> Took me a while to find the values of the default parameters, they are in nextflow.config. Maybe adding a pointer for oblivious snakemake users would be helpful :)

Do you think it would be enough if we changed row 15 to something like this?:
```
Run the pipeline with default parameters read from nextflow.config :
```
If so we may also want to update the last sentence in the [README](https://github.com/iRNA-COSI/APAeval/blob/add_working_Q2_summaryworkflow/summary_workflows/quantification/README.md#running-the-summary-workflow) accordingly:
```
One can use the following command to run the summary workflow from command line:

nextflow run main.nf -profile docker
```

Reply:
> And am I correct in assuming that the default parameter values are left untouched, but the nextflow call would be modified with the mandatory arguments? That is, a valid run on real data must always have the mandatory arguments below?

I'm not quite sure I understand what you mean here. Both runs with test and real data have to specify all the arguments. The difference is that with test data they are all specified in `nextflow.config` whereas when we run it in the OEB environment we overwrite some of the values on the command-line, which takes precedence. This is explained in the [Nextflow docs](https://www.nextflow.io/docs/latest/config.html#configuration-file).

Reply:
Yes, I think the changes you mention make sense and are sufficient.

Thanks for the explanation. I think this answers my question and I think it is a good approach to overwrite defaults. We only need to make sure we are aware of them.

Reply:
I updated the [README](https://github.com/iRNA-COSI/APAeval/blob/add_working_Q2_summaryworkflow/summary_workflows/quantification/README.md) and [main.nf](https://github.com/iRNA-COSI/APAeval/blob/add_working_Q2_summaryworkflow/summary_workflows/quantification/main.nf#L15) as we agreed.